lib/fastlane/plugin/wpmreleasetoolkit/actions/common/get_prs_list_action.rb

```diff
@@ -6,34 +6,19 @@ module Actions
     class GetPrsListAction < Action
       def self.run(params)
         repository = params[:repository]
-        start_tag = params[:start_tag]
-        end_tag = params[:end_tag]
-        report_path = params[:report_path]
+        report_path = File.expand_path(params[:report_path])
+        tag = params[:tag]
 
         # Get commit list
-        commit_list = sh("git log --pretty=oneline #{start_tag}..#{end_tag}")
+        pr_list = Fastlane::Helper::GithubHelper.get_prs_for_milestone(repository, tag)
```

Comment:
```suggestion
        pr_list = Fastlane::Helper::GithubHelper.get_prs_for_milestone(repository, milestone)
```

Reply:
Addressed in 8c3e4affab2acde6067ba4af4de8d044591a350d